lib/plugins/awsResourcesDeploy/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "serverless-aws-resources-deploy-plugin",
```

Comment:
There was a proposal to define a plugin naming convention such as `serverless-plugin-X`. If we go this way autocomplete on NPM (and search in general) for example would work way better.